### DISCUSSION TITLE
Copilot student subscription is gone?

### DISCUSSION THREAD

**Discussion by NeoHBz** [community/community#44580] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi, I am a verified GitHub student and have been using Copilot for a while now; until today the subscription broke?

The VSCode extension and the Copilot screen on GitHub says "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
PS: I am located in India.

<img width="780" alt="Screenshot 2023-01-17 at 6 41 16 PM" src="https://user-images.githubusercontent.com/56602090/212911174-3a3a93ae-af4d-483f-8ced-c8fa126e4207.png">

**Comment by softwaredevelop** [comment]
https://github.com/orgs/community/discussions/44262

My Copilot student pack account was restored after service crash.

Patience, good luck and try to GitHub support team.

**Comment by tiany7** [comment]
Me too, the renewed access ends on 2023-2-8 after the latest service crash. However, today it suddenly stopped working again. Hope this could be resolved asap.

**Comment by mafqla** [comment]
我的也是最近一直跳出结束试用，点击付费又跳转到学生包了。
